kube-client/src/config/file_config.rs

```diff
@@ -519,12 +542,12 @@ users:
     client-certificate: /home/kevin/.minikube/profiles/minikube/client.crt
     client-key: /home/kevin/.minikube/profiles/minikube/client.key";
 
-        let config: Kubeconfig = serde_yaml::from_str(config_yaml).unwrap();
+        let config = kubeconfig_from_yaml(config_yaml).unwrap();
```

Comment:
can we test the public interface here instead: `Kubeconfig::from_yaml`?

Reply:
No problem